pkg/test/framework/components/cluster/cluster.go

```diff
@@ -102,6 +102,13 @@ func (c Clusters) Remotes(excluded ...Cluster) Clusters {
 	}, exclude(excluded...))
 }
 
+// MeshClusters returns the subset that are not external control plane clusters.
+func (c Clusters) MeshClusters(excluded ...Cluster) Clusters {
+	return c.filterClusters(func(cc Cluster) bool {
+		return !cc.IsExternalControlPlane()
```

Comment:
Should we also have an `isMeshCluster` on the singular `Cluster` interface?

Reply:
I would leave it for now, since it's just the inverse of `IsExternalControlPlane`. In the future, if we come up with other kinds of external clusters, we can add it.